wiki/Ranking_Criteria/Skinning/en.md

```diff
@@ -0,0 +1,331 @@
+Skin Set List 
+============= 
+
+The following table contains all skin sets that are presented in the draft worked out by the skinning subdivision of the [UBKRC-Project](https://osu.ppy.sh/forum/t/480331) (name credits go to Ephemeral)
+
+## osu! Cursor Set
+
+The entirety of this skinset is gameplay relevant.
+
+|File Name   |Notes   |Set Completion   |Recommended Sizes for SD Skins   |
+|--:|:-:|:-:|:-:|
+|cursor.png   |should always be included together with cursortrail.png   | required   | -  |
+|cursortrail.png   |should always be included together with cursor.png   | required  | -  |
+|cursormiddle.png   |should be optional depending on the cursor's design   | optional  | -   |
+|cursor-smoke.png   |-   | optional   | -   |
+
+## osu! Hitburst Set
+
+The entirety of this skinset is gameplay relevant.
+
+|File Name   |Notes   |Set Completion   |Recommended Sizes for SD Skins   |
+|--:|:-:|:-:|:-:|
+|hit0(-n).png   | should be clearly distinguishable from all other hitbursts  | required  | -  |
+|hit50(-n).png   | should be clearly distinguishable from all other hitbursts  | required  | -  |
+|hit100(-n).png   | should be clearly distinguishable from all other hitbursts  | required  | -  |
+|hit100k(-n).png   | should be clearly distinguishable from all other hitbursts  | required  | -  |
+|hit300(-n).png   | should be clearly distinguishable from all other hitbursts   | required  | -  |
+|hit300g(-n).png   | should be clearly distinguishable from all other hitbursts  | required  | -  |
+|hit300k(-n).png   | should be clearly distinguishable from all other hitbursts  | required  | -  |
+|lighting.png   | -  | optional  | -  |
+|particle50.png   | should be included together with all other particles; will only work if custom hitbursts are included   | optional  | 7x7  |
```

Comment:
I don't know how repetitive you want to be, but most of the dimensions listed don't contain units.
e.g. `11x17` should be `11x17px` 🤔